### DISCUSSION TITLE
Create history for how Principles have evolved

### DISCUSSION THREAD

**Discussion by adamj89** [Green-Software-Foundation/learn#193] [General]
Discussion from Slack:
https://greensoftwarefdn.slack.com/archives/C023N44UGN8/p1685646273243819

Potential to add a history for how the principles have evolved.

**Comment by cg122** [comment]
Also there are 6 principles described on the [introduction](https://learn.greensoftware.foundation/introduction) page, but only 3 principles [depicted](https://learn.greensoftware.foundation/assets/images/01_carbon_efficiency-e5ce9cc2af3c13c4154ff61a0267b2b5.png)  on the same page. It is a bit confusing.
